transcoder/message/MessageUtil.py

```diff
@@ -46,3 +46,28 @@ def get_message_parser(factory: str, schema_file_path: str,
         raise MessageParserNotDefinedError
 
     return message_parser
+
+
+def parse_handler_config(handler_config_string: str) -> dict:
+    """ 
+    Extracts the configuration parameters attached to the CLI handler option, 
+    in the format:
+
+    FirstHandler:<param>=<value>,SecondHandler:<param>=<value>
+
+    For example:
+
+    --message_handlers SequencerHandler,FilterHandler:field=value
+
+    would run a SequencerHandler without a config, then pass a single-element dict of field: value to FilterHandler via the config object.
+
+    This routine manufactures the configuration for the Handler from the CLI string
+
+    """
+    if handler_config_string.find(':') != -1:
```

Comment:
@salsferrazza - This is only supporting one parameter per handler specified?

Reply:
for this particular CL yes, but absolutely would like to discuss how to encode a larger dictionary of options within the CLI. It's mostly about determining `MessageHandler`'s parameter delimiter format such that it's CLI friendly. For example. we might consider using `&` (a la HTTP query strings) but since that is a special shell character, it may require escaping or similar tactic to be shell friendly.